From dee8ba21efaa87f106153c73bb18f60f39aa1f5b Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Fri, 20 Jun 2025 16:25:50 -0400
Subject: Add "&& !(defined(__CYGWIN__)" after defined(_WIN32)

---
 libtiff/tif_unix.c | 2 +-
 libtiff/tiffiop.h  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libtiff/tif_unix.c b/libtiff/tif_unix.c
index 9e6fc75b..fce4e354 100644
--- a/libtiff/tif_unix.c
+++ b/libtiff/tif_unix.c
@@ -270,7 +270,7 @@ TIFF *TIFFOpenExt(const char *name, const char *mode, TIFFOpenOptions *opts)
     return tif;
 }
 
-#ifdef _WIN32
+#if defined(_WIN32) && !(defined(__CYGWIN__))
 #include <windows.h>
 /*
  * Open a TIFF file with a Unicode filename, for read/writing.
diff --git a/libtiff/tiffiop.h b/libtiff/tiffiop.h
index 80c04c27..1858a096 100644
--- a/libtiff/tiffiop.h
+++ b/libtiff/tiffiop.h
@@ -358,7 +358,7 @@ struct TIFFOpenOptions
 #define ftell(stream, offset, whence) ftello(stream, offset, whence)
 #endif
 #endif
-#if defined(_WIN32) &&                                                         \
+#if defined(_WIN32) && !(defined(__CYGWIN__)) &&                               \
     !(defined(__MSVCRT_VERSION__) && __MSVCRT_VERSION__ < 0x800)
 typedef unsigned int TIFFIOSize_t;
 #define _TIFF_lseek_f(fildes, offset, whence)                                  \
-- 
