From 0eb3cfa329f23e23098b1f29f4e17e1f21251f7e Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Mon, 3 Aug 2026 19:14:10 -0400
Subject: Msys: Add __MSYS__ guard

---
 src/glew.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/glew.c b/src/glew.c
index f7b64cf..7f8eebd 100644
--- a/src/glew.c
+++ b/src/glew.c
@@ -21671,7 +21671,7 @@ GLenum eglewInit (EGLDisplay display)
   return GLEW_OK;
 }
 
-#elif defined(_WIN32)
+#elif defined(_WIN32) || defined(__MSYS__)
 
 PFNWGLSETSTEREOEMITTERSTATE3DLPROC __wglewSetStereoEmitterState3DL = NULL;
 
-- 
2.53.0.windows.2

