From 5e62f8f4949aecff020f217239b12ceedf911a1f Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sat, 8 Aug 2026 06:46:16 -0400
Subject: Cygwin: Change DLL name

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b243a8070..5e68fd2ce 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3988,13 +3988,13 @@ if(SDL_SHARED)
         RESOURCE "${SDL_FRAMEWORK_RESOURCES}"
       )
     endif()
-  elseif(UNIX AND NOT ANDROID)
+  elseif(UNIX AND NOT (ANDROID OR CYGWIN))
     set_target_properties(SDL3-shared PROPERTIES
       VERSION "${SDL_SO_VERSION}"
       SOVERSION "${SDL_SO_VERSION_MAJOR}"
     )
   else()
-    if(WINDOWS OR CYGWIN)
+    if(WINDOWS)
       set_target_properties(SDL3-shared PROPERTIES
         PREFIX ""
       )
-- 
2.53.0.windows.2

